modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -134,6 +148,8 @@ public void registerExtension(ExtensionModel extensionModel) {
       LOGGER.debug("Registering extension {} (version: {} vendor: {} )", extensionName, extensionVersion, extensionVendor);
     }
 
+    extensionJdkValidator.validateJdkSupport(extensionModel);
```

Comment:
instead of failing on the first extension found, we should log a single failure with all unsupported extensions

Reply:
--validate